fuzzers/029-bram-fifo-config/generate.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import json
+
+from prjxray.segmaker import Segmaker
+
+
+def bitfilter(frame, bit):
+    # Filter out interconnect bits.
+    if frame in [0, 1, 17, 23, 25]:
```

Comment:
Are these reference frame numbers fixed or only artix7 specific? More in general, where do they come from?

Reply:
So far, the bits for a7, k7 and z7 have been the same, so this is likely not a7 specific, but only testing will give us confidence.

As for where the frame numbers come from:
https://github.com/SymbiFlow/prjxray-db/blob/master/artix7/segbits_int_l.db
https://github.com/SymbiFlow/prjxray-db/blob/master/kintex7/segbits_int_l.db